packages/design-system/src/Toast/Toast.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { useFocusRing } from "@react-aria/focus";
+import { Slide, toast as toastifyToast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.min.css";
+
+import { Kind, ToastProps } from "./Toast.types";
+import { StyledToast, ToastBody } from "./Toast.styles";
+import { Icon } from "../Icon";
+
+/**
+ * TODO:
+ * - Add a variant that carries a link
+ * - Add a way to attach an action to that link
+ * - Look up accessibility features for toasts (spectrum, MDN, what toastify does and does not support)
+ * - Add a control for the content of toast in the story
+ * @constructor
+ */
+
+function Toast({ ...rest }: ToastProps) {
+  return (
+    <StyledToast
+      autoClose={5000}
+      className="t--toast-action"
```

Comment:
Why this class is hardcoded? `t--` is used to denote test classnames. For ads components used ads--

Reply:
don't we need to add the cypress test names to our components to ensure they don't break existing tests?

Reply:
That should be passed from main repo

Reply:
We don't keep any those kind of classnames in DS